Dockerfile-windows.template

```diff
@@ -0,0 +1,57 @@
+FROM mcr.microsoft.com/windows/servercore:version as installer
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# PATH isn't actually set in the Docker image, so we have to set it from within the container
+RUN $newPath = ('C:\Program Files (x86)\GnuPG\bin;{0}' -f $env:PATH); \
+    Write-Host ('Updating PATH: {0}' -f $newPath); \
+    [Environment]::SetEnvironmentVariable('PATH', $newPath, [EnvironmentVariableTarget]::Machine)
+# doing this first to share cache across versions more aggressively
+
+ENV NODE_VERSION 0.0.0
+ENV NODE_CHECKSUM CHECKSUM_x64
```

Comment:
Shouldn't this value be used somewhere below?

Reply:
Not really, this is just leftover from when I started working on this, as I tried multiple combinations compiled from other PRs. Will remove that.

Reply:
I would much rather see it get used, TBH - we have the data, we might as well verify it (especially since it's really trivial to do so)

Reply:
Yeah, you're right, it will make it simpler, and we don't need to use multistage for servercore. That is, I still think that it will be much easier to use servercore as an installer for nanoserver if we decide to add it sometime down the road.